### PR TITLE
Role Added in Form

### PR DESCRIPTION
**Pull request by anupkore**

### CODE REVIEW

File: src/Components/AddNewMember.js [hunk 1]
```diff
@@ -139,6 +141,28 @@ export default function AddNewMember()
...
+                >
+                    <option value="">Select Role</option>
+                    <option  value="Manager">
+                        Project Manager
+                    </option>
+                     <option  value="User">
+                        Team Member
+                     </option>
+                </select>
+
```
Comment (YJB28) [inline src/Components/AddNewMember.js:161]: remove Unnecessary Spaces.

**Review by Yashwant-0108 — APPROVED** [review]
Checked

**Review by YJB28 — APPROVED** [review]
Remove spaces and merge. Code run successfully.

**Comment by anupkore** [comment]
Spaces Removed...Checked

**Pull request merged**